README.md

```diff
@@ -115,7 +155,7 @@ Describe what a schema is and how we represent a one-to-many relationship in a
 SQL database.
 
 ```
-Your answer goes here...
+
 ```
```

Comment:
A schema specifies what information goes into a table, how that table is structured.